### ISSUE TITLE
[Table] Phase 1 Feature - Entity Group Transactions (Batch)

### ISSUE THREAD

**Issue by XiaoningLiu** [Azure/Azurite#493]
Read through:

- https://docs.microsoft.com/en-us/rest/api/storageservices/performing-entity-group-transactions
- [Azurite V3 Table wiki](https://github.com/Azure/Azurite/wiki/Azurite-V3-Table)

Implement following parts:

- [ ] Check swagger has $batch RESTful defined. If not, add it into Azurite swagger [transform](link), then regenerate
- [ ] Implement TableHandler::batch()
- [ ] Implement metadata layer method in LokiTableMetadataStore if needed
- [ ] Add test cases

Take Create Table implementation as example:
- [TableHandler::create()](link)
- [LokiTableMetadataStore::createTable()](link)
- [Create Table test cases](link)

**Comment by hemisphera** [comment]
Also in regard to #264. Any update or planned release date on this? Now that the Azure Storage Emulator has officially been labelled as obsolete/deprecated I have tried to switch to Azurite but without batch operations on table storage it is unusable for me. 

I can see that a milestone was added on july 7, but no due date yet.

**Comment by edwin-huber** [comment]
Hi, 
the main branch already supports batch, but we are still working on complete feature parity.
Issues which need to be completed before we announce GA on Table are listed there, and you can follow our progress against that list.

**Comment by edwin-huber** [comment]
Batch API and transaction rollback are now merged into main, please expect this in the next release.
